### PR TITLE
Update doc for load balancing with message processor

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Purpose
[Update doc for load balancing with message processor](https://github.com/wso2/docs-mi/commit/9af5434147093fa08164140127ccf10374be52ab)

### CODE REVIEW

File: en/docs/learn/examples/message-store-processor-examples/loadbalancing-with-message-processor.md [hunk 1]
```diff
@@ -86,21 +86,54 @@ Following are the artifact configurations that we can use to implement this scen
...
+   ```
+   === "On Windows"          
+   ```bash
+   axis2server.bat
+   ```
 
-You can analyze the message sent by the Micro Integrator to the secure service using TCPMon.
+Send the following request to invoke the service:
 
-On successful execution of the placeorder request, you will see the following message on the back-end:
```
Comment (chanikag) [inline en/docs/learn/examples/message-store-processor-examples/loadbalancing-with-message-processor.md:100]: Why is this content removed?

Comment (kalaiyarasiganeshalingam) [reply en/docs/learn/examples/message-store-processor-examples/loadbalancing-with-message-processor.md:100]: This doc doesn't have a sample request. So, I updated those things. 

Here, we mentioned this [here](https://github.com/wso2/docs-mi/pull/388/files#diff-ee2fb1d2853c20da213130764044b4ad008548a26d83ed8408c705958fcc89ceR139)

I got a different log in the back-end server when I tested it. So, I removed it.

Comment (chanikag) [reply en/docs/learn/examples/message-store-processor-examples/loadbalancing-with-message-processor.md:100]: what is the log you got in the back-end server?

Comment (kalaiyarasiganeshalingam) [reply en/docs/learn/examples/message-store-processor-examples/loadbalancing-with-message-processor.md:100]: Mon Aug 12 13:19:27 IST 2024 samples.services.SimpleStockQuoteService :: Generating quote for : IBM

**Review by chanikag — APPROVED** [review]

**Pull request merged**